sqlcollaborative/dbatools
# Get-DbaSsisEnvironment

Fixes # none

New function. Previous PR closed, this one uses functions mentioned during the reviev, uses foreach loops, objects, exclusions, Silence swith and probably some other neat things. Previously I put two functions in the PR, this time it's first of them, second on the way.

Changes proposed in this pull request:
 - Get-DbaSsisEnvironment - gets SSIS environment variables, even if encrypted

How to test this code: 
- see EXAMPLES; also I have Pester integration tests, but not sure how to prepare the setup for the tests.

Has been tested on minimum requirements:
- [ ]  Powershell 3 - no (4)
- [ ]  Windows 7 - no (Windows Server 2012R2)
- [ ]  SQL Server 2000 - no (minimum required SQL Server 2012 - SSIS Catalog was introduced then)

Has been tested on maximum requirements:
- [ ]  SQL Server vNext - no
- [ ]  Windows 10 - no
- [ ]  Azure Database - no

Tests for tester:
